util/src/main/java/tech/pegasys/artemis/util/config/ArtemisConfiguration.java

```diff
@@ -187,17 +184,16 @@ static final Schema createSchema() {
     builder.addInteger("constants.PERSISTENT_COMMITTEE_PERIOD", Integer.MIN_VALUE, null, null);
     builder.addInteger("constants.MAX_EPOCHS_PER_CROSSLINK", Integer.MIN_VALUE, null, null);
     builder.addInteger("constants.MIN_EPOCHS_TO_INACTIVITY_PENALTY", Integer.MIN_VALUE, null, null);
-    builder.addInteger(
-        "constants.EARLY_DERIVED_SECRET_PENALTY_MAX_FUTURE_EPOCHS", Integer.MIN_VALUE, null, null);
 
     // State list lengths
-    builder.addInteger("constants.LATEST_RANDAO_MIXES_LENGTH", Integer.MIN_VALUE, null, null);
-    builder.addInteger("constants.LATEST_ACTIVE_INDEX_ROOTS_LENGTH", Integer.MIN_VALUE, null, null);
-    builder.addInteger("constants.LATEST_SLASHED_EXIT_LENGTH", Integer.MIN_VALUE, null, null);
+    builder.addInteger("constants.EPOCHS_PER_HISTORICAL_VECTOR", Integer.MIN_VALUE, null, null);
+    builder.addInteger("constants.EPOCHS_PER_SLASHINGS_VECTOR", Integer.MIN_VALUE, null, null);
+    builder.addInteger("constants.HISTORICAL_ROOTS_LIMIT", Integer.MIN_VALUE, null, null);
+    builder.addLong("constants.VALIDATOR_REGISTRY_LIMIT", Long.MIN_VALUE, null, null);
```

Comment:
Sanity check note: Figured I'd double check Tuweni SchemaBuilder#addLong would support a value as big as VALIDATOR_REGISTRY_LIMIT (1,099,511,627,776) and it does, since that's well within the bounds of a signed Long.